charts/xrd-vrouter/values.yaml

```
@@ -256,6 +255,16 @@ cpu: {}
   #               * physical core 1 has logical cores 2 and 3
   #               * etc...
   #hyperThreadingMode: "off"
+  #
+  # Deprecated cpu settings:
+  #
+  # Override the cpuset to use for XRd. Must not be used in conjunction with
+  # controlPlaneCpuset and dataPlaneCpuset. (Deprecated.)
+  # This must be a subset of the CPUs available for XRd to run on.
+  #cpuset: ""
+  # Override the number of CPUs used for control-plane threads from the pool of
+  # CPUs in the XRd cpuset, used in conjunction with cpuset. (Deprecated.)
```

```suggestion
  # CPUs in the XRd cpuset. (Deprecated.)
```

This doesn't have to be specified if cpuset is (and vice-versa AIUI), so this line is misleading.

This should be the same warning as for cpuset - "Must not be used in conjunction with controlPlaneCpuset and dataPlaneCpuset".

Alternatively I think it's probably fine to omit this entirely and stick with the old documentation if you prefer, given that this is marked as deprecated.

---

This should also point the user towards controlPlaneCpuset and dataPlaneCpuset as an alternative